packages/vue/src/components/organisms/SfHeader/_internal/SfHeaderNavigationItem.vue

```diff
@@ -1,10 +1,40 @@
 <template>
   <div class="sf-header-navigation-item" v-on="$listeners">
-    <slot />
+    <div class="sf-header-navigation-item--desktop">
+      <SfLink class="sf-header-navigation-item__link" :link="link">{{
+        label
+      }}</SfLink>
+      <slot />
```

Comment:
Isn't it better to put SfLink inside slot? Currently there are two links when you put some link inside slot linke in MegaMenu story:  
![image](https://user-images.githubusercontent.com/32042425/86594519-74470600-bf97-11ea-89cd-27bfc2899142.png)

Reply:
I added a separate slot for it :)